tests/manual-test-cases/Group18-VIC-UI/18-3-VIC-UI-NGC-tests.robot

```diff
@@ -36,17 +39,21 @@ Check Prerequisites
 
 Ensure Vicui Is Installed
     # ensure vicui is installed before running ngc automation tests
-    Set Vcenter Ip
-    Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    #Install Vicui Without Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}  ${TEST_VC_ROOT_PASSWORD}  ${TRUE}
+    Update Config To Use Webserver For Plugin Testing
+    Install With Webserver  ${TEST_VC_USERNAME}  ${TEST_VC_PASSWORD}
     ${output}=  OperatingSystem.GetFile  install.log
-    Should Contain  ${output}  was successful
+    ${passed}=  Run Keyword And Return Status  Should Contain  ${output}  was successful
+    Run Keyword Unless  ${passed}  Copy File  install.log  ngc-tests-fail-install-plugin-using-webserver-for-plugin-testing.log
+    Remove File  install.log
+    Should Be True  ${passed}
```

Comment:
This whole section is kinda weird...
1. maybe just move instead of copy, then remove?
2. you are just trying to ensure that the install succeeded right? this seems like a lot of extra steps that are more than that
3. Install with webserver should really just ensure that the install was successful within it.